spaces/S000110/properties/P000132.md

```diff
@@ -0,0 +1,11 @@
+---
+space: S000110
+property: P000132
+value: true
+---
+
+Let $K \subset X$. Since all free ultrafilters on $\omega$ contain all cofinite subsets of $\omega$, we see that,
```

Comment:
```suggestion
Let $K \subset X$. If $K \subset \omega$, then it is already open. Otherwise, since all free ultrafilters on $\omega$ contain all cofinite subsets of $\omega$, we see that,
```